### PR TITLE
[#176038005] method history

### PR DESCRIPTION
**Pull request by vito80ba**
Now there is a specific component to call the REST service to display the "history" of a payment method

### CODE REVIEW

File: .env.local
Comment (gunzip) [inline .env.local:1]: of course this must change before merging :-)

File: azure-pipelines.yml
Comment (gunzip) [inline azure-pipelines.yml:36]: can we keep these ones as pipelines variables ? in order to avoid to commit them

Comment (gunzip) [reply azure-pipelines.yml:36]: these changes are unrelated btw and must go in another pr

File: src/helpers/msal.ts
Comment (gunzip) [inline src/helpers/msal.ts:29]: move this into another pr

**Review by gunzip — CHANGES_REQUESTED** [review]

File: .env.local
Comment (vito80ba) [reply .env.local:1]: env.local file is for "local development"... where is the problem? :)

File: azure-pipelines.yml
Comment (vito80ba) [reply azure-pipelines.yml:36]: These changes are yet in another PR (rebase when I made this one)
https://github.com/pagopa/backoffice.io.italia.it/pull/28 (waiting for approval)

**Review by pasqualedevita — COMMENTED** [review]
Can we add also `channel` and `channel_descr`?
https://github.com/pagopa/io-functions-backoffice/pull/33
`channel_descr` must to be visible on frontend
`channel` visible only in raw mode

**Pull request merged**
